neuralmonkey/learning_utils.py

```diff
@@ -248,6 +251,26 @@ def argworst(scores, minimize):
     log("Finished.")
 
 
+def _check_series_collisions(runners: List[BaseRunner],
```

Comment:
Mně se nechce a to ze dvou důvodů: (1) v modulu `checking` jsou aserce, co se používají na více místech kódu a (2) v modulu checking by nešlo reusovat definici typu `Postprocess` z learning utils (jinak bysme měli cyklický dependnce) a stěhovat `Postprocess` do `checking` je blbost.